apps/ensnode/src/handlers/Registrar.ts

```diff
@@ -94,6 +102,7 @@ export const makeRegistrarHandlers = (ownedName: OwnedName) => {
         registrantId: owner,
         expiryDate: expires + GRACE_PERIOD_SECONDS,
         labelName,
+        name,
```

Comment:
@shrugs & I have discussed adding this property to follow the ENS Subgraph logic more accurately:
https://github.com/ensdomains/ens-subgraph/blob/c8447914e8743671fb4b20cffe5a0a97020b3cee/src/ethRegistrar.ts#L59